vividus-plugin-rest-api/src/test/java/org/vividus/bdd/steps/api/HttpRequestStepsTests.java

```diff
@@ -149,6 +156,14 @@ void testSetCustomRequestConfigNoField()
         assertEquals("nonExistentField", exception.getMessage());
     }
 
+    @Test
+    void testWaitForResponse() throws IOException
+    {
+        httpRequestSteps.waitForResponse(RESPONSE_CODE, HttpMethod.GET, URL, DURATION, RETRY_TIMES);
+        verify(httpRequestExecutor).executeHttpRequest(eq(HttpMethod.GET), eq(URL), eq(Optional.empty()),
+                any(), any(WaitMode.class));
```

Comment:
do not use `any` to verify expected successful invocations, use `eq`, `argThat` and etc.